content/v2.0/actions/overview.md

```diff
@@ -45,7 +45,7 @@ Topics to cover:
 - response headers
 - redirects
 - halt
-- HTTP caching
+- exception handling
```

Comment:
Don't think you meant to add this back?

Reply:
nope, accidental

Reply:
I've removed that fully from the main branch now; just delete it here too IMO

Reply:
(via the "Resolve conflicts" button — just be sure to "Squash and merge" afterwards)